modules/gitops-release-notes-1-4-5.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assembly:
+//
+// * gitops/gitops-release-notes.adoc
+
+[id="gitops-release-notes-1-4-5_{context}"]
+= Release notes for {gitops-title} 1.4.5
+
+[role="_abstract"]
+{gitops-title} 1.4.5 is now available on {product-title} 4.7, 4.8, and 4.9.
+
+[id="fixed-issues-1-4-5_{context}"]
+== Fixed issues
+
+The following issue has been resolved in the current release:
+
+
+[WARNING]
```

Comment:
Consider moving the "Warning" admonition up above the intro sentence (so between `== Fixed issues` and `The following issue has been resolved in the current release:`) sentence because the admonition applies generally and not just to the specific issue you mention.